deployment/configutils/CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ include_directories (
          ${HPCC_SOURCE_DIR}/deployment/deploy
          ${CMAKE_BINARY_DIR}/oss
          ${CMAKE_BINARY_DIR}
+         ${HPCC_SOURCE_DIR}/system/security/shared
```

Comment:
Why do we use relative paths in some cases? (deployment/configgen/CMakeLists.txt line 38 for example)